packages/test-utils/src/mocks.ts

```diff
@@ -24,6 +24,11 @@ export class MockedServer {
     this.clearRequests()
   }
 
+  shouldHaveLastRequestMatchSnapshot (name?: string) {
```

Comment:
Naming is so hard......

Reply:
> Naming is so hard......

Pas aussi dur que l'anglais et le chinois...